corehq/apps/hqmedia/views.py

```diff
@@ -192,6 +192,8 @@ def post(self, request, *args, **kwargs):
             response.update(self.process_upload())
         except BadMediaFileException as e:
             self.errors.append(e.message)
+        except IOError:
+            self.errors.append(_('Upload is not a valid image file.'))
```

Comment:
Is this really only raised for images and not audio/video?

Reply:
Oddly enough it's only images

Reply:
It's only images because it's coming from PIL (image processing library). If possible it might be better to add this to `CommCareImage` and re-raise as `BadMediaFileException`

Reply:
good point - updated